sdk/scheduler/src/test/java/com/mesosphere/sdk/state/StateStoreTest.java

```diff
@@ -20,7 +20,7 @@
 /**
  * Tests to validate the operation of the {@link StateStore}.
  */
-public class DefaultStateStoreTest {
+public class StateStoreTest {
     private static final Protos.FrameworkID FRAMEWORK_ID =
             Protos.FrameworkID.newBuilder().setValue("test-framework-id").build();
     private static final String TASK_NAME = "test-task-name";
```

Comment:
Isn't this `TASK_NAME` a duplicate of `TestConstants.TASK_NAME`? What is the motivation for using static members with the same names from `TestConstants` in general?